src/components/AddWorkout.jsx

```diff
@@ -0,0 +1,116 @@
+import { useEffect } from "react";
+import axios from "axios";
+import { useState } from "react";
+import { Paper } from "@mui/material";
+import { paperStyles } from "../styles/styles";
+import AddIcon from '@mui/icons-material/Add';
+import { IconButton } from "@mui/material";
+import Dialog from '@mui/material/Dialog';
+import DialogTitle from '@mui/material/DialogTitle';
+import DialogContent from '@mui/material/DialogContent';
+import DialogActions from '@mui/material/DialogActions';
+import CloseIcon from '@mui/icons-material/Close';
+import { styled } from '@mui/material/styles';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+
+
+
+
+function AddWorkout() {
+// workout is an array of exercises
+  const [workout, setWorkout] = useState([]);
+  const [dialogOpen, setDialogOpen] = useState(false);
+
+    let data, filterMethod = "muscle", filterType = "biceps" 
+
+    useEffect(() => {
+        axios.get('https://api.api-ninjas.com/v1/exercises'+`?${filterMethod}=${filterType}`, {headers: { 'X-Api-Key': '/2l4M9V5VpBLETtKJzc5Mw==mPTtJXrI5qzXquuf' }})
+        .then((response) => {
+            data = response.data
+            setWorkout([...workout, data[0]])
+        })
+        .catch((error) => {console.log(error)})
+    }, [])
+
+    const handleClickOpen = () => {
+        setDialogOpen(true);
+    }
+
+    const handleClickClose = () => {
+        setDialogOpen(false);
+    }
+
+    return ( 
+        <>
+
+            {workout && workout.map((exercise, i) => {
+                return(
+                    <div key={i}>
```

Comment:
do you need the div ? does it work the same if you remove the div and put the key in the paper component ?